beets/dbcore/query.py

```diff
@@ -447,6 +447,34 @@ def match(self, item):
         return any([q.match(item) for q in self.subqueries])
 
 
+class NotQuery(Query):
+    """A query that matches the negation of its `subquery`, as a shorcut for
+    performing `not(subquery)` without using regular expressions."""
+    def __init__(self, subquery):
+        self.subquery = subquery
+
+    def clause(self):
+        clause, subvals = self.subquery.clause()
+        if clause:
+            return 'not ({0})'.format(clause), subvals
+        else:
+            # special case for RegexpQuery, (None, ())
```

Comment:
In fact, this is actually the case for any "slow" query. For explicitness, I might suggest putting this in `__init__`:

    self.fast = subquery.fast

And then here:

    if self.fast:
        clause, subvals = ...
        return ...
    else:
        # A slow query.
        return None, ()

as in the `clause` method of `FieldQuery`, for example.